APP-94627 pendo web (actions) doc (#5866)
segmentio/segment-docs: src/connections/destinations/catalog/pendo-web-actions/index.md

---

> title: Pendo Web (Actions) Destination

Reviewer: For context: the `id` is generated by Segment's Public API - including it here autogenerates info like the actions-fields, and adds this page to the [destination catalog page](https://segment.com/docs/connections/destinations/catalog/). The `hidden` and `beta` attributes denote that this is in private beta. 

```suggestion
title: Pendo Web (Actions) Destination
id: 6501a4325a8a629197cdd691
hidden: true
beta: true
```

<!-- In the section below, add your destination name where indicated. If you have a classic version of the destination, ensure that its documentation is linked as well. If you don't have a classic version of the destination, remove the second and third sentences. -->

> success ""
> **Good to know**: This page is about the [Actions-framework](/docs/connections/destinations/actions/) Pendo Web (Actions) Segment destination. There's also a page about the [non-Actions Pendo destination](/docs/connections/destinations/catalog/pendo/). Both of these destinations receives data from Segment.

<!-- This include describes the requirement of A.js 2.0 or higher for Actions compatibility, and is required if your destination has a web component. -->

{% include content/ajs-upgrade.md %}

<!-- In the section below, explain the value of this actions-based destination over the classic version, if applicable. If you don't have a classic version of the destination, remove this section. -->

## Benefits of Pendo Web (Actions) vs Pendo Classic

Pendo Web (Actions) provides the following benefits over the classic Pendo destination:

> - **Support for `track`, `identify`, and `group`**. The classic Pendo destination did not support `track`, and required an additional webhook destination be configured.

Reviewer: ```suggestion
- **Support for Track, Identify, and Group calls**. The classic Pendo destination did not support Track calls and required users to configure an additional Webhook destination.
```

- **Full region support**. Setup allows the destination to be configured for US, EU, US restricted, or Japan.

> - **Supports CNAME for Pendo**.  Works with subscriptions using Pendo's custom CNAME feature.

Reviewer: ```suggestion
- **Supports CNAME for Pendo**. Works with subscriptions using Pendo's custom CNAME feature.
```

## Getting started

1. From the Segment web app, click **Catalog**, then click **Destinations**.
2. Find the Destinations Actions item in the left navigation, and click it.
3. Click **Pendo Web (Actions)**.
4. Click **Add destination ->**.
5. Select an existing Source to connect to Pendo Web (Actions).
6. In the destination settings, enter your Pendo API Key which you can find in the Pendo UI under [Site Settings](https://app.pendo.io/admin){:target="_blank"} > Basic Information > API Key.
7. Select the correct region for your Pendo subscription.

<!-- The line below renders a table of connection settings (if applicable), Pre-built Mappings, and available actions. -->

{% include components/actions-fields.html %}

<!--
Additional Context

Include additional information that you think will be useful to the user here. For information that is specific to an individual mapping, please add that as a comment so that the Segment docs team can include it in the auto-generated content for that mapping.
-->

<!-- If applicable, add information regarding the migration from a classic destination to an Actions-based version below -->

## Migration from the classic Pendo destination

> Remove the classic Pendo destination and webhook destination from your Segment account before adding the Pendo Web (Actions) destination.

Reviewer: ```suggestion
Remove the classic Pendo destination and webhook destination from your Segment workspace before adding the Pendo Web (Actions) destination.
```
